### PR TITLE
Deployment fix

### PR DESCRIPTION
**Pull request by KennethSkylight**
All the changes needed to fix the deployment. Removed data factory e2e test. Remove storage references, decreased the node count. 

It deploys the helm charts into AKS using terraform

### CODE REVIEW

File: .github/workflows/destroy.yaml [hunk 1]
```diff
@@ -59,7 +59,6 @@ jobs:
           echo object_id=\""$OBJECT_ID"\" >> terraform.tfvars
           echo use_oidc=true >> terraform.tfvars
           echo resource_group_name=\""$RESOURCE_GROUP_NAME"\" >> backend.tfvars
-          echo storage_account_name=\"phditfstate"${CLIENT_ID:0:8}"\" >> backend.tfvars
```
Comment (nickclyde) [inline .github/workflows/destroy.yaml:62]: Not that we ever use the destroy workflow, but I think we actually still need this line there

File: .github/workflows/end-to-end.yaml [hunk 2]
```diff
@@ -30,61 +30,4 @@ jobs:
...
-              exit 1
-            fi
-            echo "Waiting for pipeline to complete run..."
-            sleep 10
-            CHECK_COUNT=$((CHECK_COUNT+1))
-          done
-
-          echo "Pipeline run succeeded!"
\ No newline at end of file
+        run: az config set defaults.location=$LOCATION defaults.group=$RESOURCE_GROUP_NAME
```
Comment (nickclyde) [inline .github/workflows/end-to-end.yaml:33]: Can you just remove this file completely, and then remove the call to it in the deployment workflow?

**Review by nickclyde — CHANGES_REQUESTED** [review]
Great work Kenneth! Just a few notes

File: .github/workflows/destroy.yaml (hunk 1 above)
Comment (KennethSkylight) [reply .github/workflows/destroy.yaml:62]: added back

**Review by nickclyde — APPROVED** [review]
Thanks for the quick changes! Looks great

**Pull request merged**